src/Components/creatorPage/AuthorInfo.js

```diff
@@ -0,0 +1,37 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import './Author.css';
+
+// Author name/bio/picture/link components
+export default function AuthorInfo({
+  author, bio, links, authorPic,
+}) {
+  return (
+    <>
+      <div className="Header">
+        {author}
+        <img width="10%" height="10%" align="center" src={authorPic} alt="author" />
+      </div>
+      <div className="Paragraph">
+        {bio}
+      </div>
+      <br />
+      <div className="Extra">
+        Check out
+        {' '}
+        {author}
+        &apos;s personal website:
+      </div>
+      <div>
```

Comment:
As this div is not applying any styling, I don't think you really need it. Instead you can leave the link tag unwrapped!